cmd/lncli/cmd_pay.go

```diff
@@ -336,7 +343,13 @@ func sendPaymentRequest(ctx *cli.Context,
 	}
 
 	req.CltvLimit = int32(ctx.Int(cltvLimitFlag.Name))
-	req.TimeoutSeconds = paymentTimeoutSeconds
+
+	pmtTimeout := ctx.Duration("timeout")
+	if pmtTimeout == 0 {
+		return errors.New("payment timeout must be non-zero")
+	}
+
+	req.TimeoutSeconds = int32(pmtTimeout.Seconds())
```

Comment:
nit: extra new line after this line

Reply:
preexisting, there are newlines between `AllowSelfPayment` and `MaxParts` below as well.